src/connector/src/sink/kafka.rs

```diff
@@ -557,8 +557,12 @@ impl KafkaTransactionConductor {
             if config.use_transaction {
                 c.set("transactional.id", identifier); // required by kafka transaction
             }
-            let client_ctx =
-                PrivateLinkProducerContext::new(config.common.broker_rewrite_map.clone())?;
+            // todo: allow collect metric here
```

Comment:
What does this mean?👀

Reply:
`stats` serves as a callback func registered in kafka config context, we need to make matrics part of the context.